lang/en/docs/cli/unlink.md

```diff
@@ -4,9 +4,11 @@ guide: docs_cli
 layout: guide
 ---
 
+{% include vars.html %}
+
 <p class="lead">Unlink a previously created symlink for a package.</p>
 
-To remove a symlinked package created with [`yarn link`](./link), `yarn unlink` can be used.
+To remove a symlinked package created with [`yarn link`]({{url_base}}/docs/cli/link), `yarn unlink` can be used.
```

Comment:
For me it doesn't work if you open https://yarnpkg.com/en/docs/cli/unlink/. Because this link is a relative one as described in the description, it will just get appended and then look like this https://yarnpkg.com/en/docs/cli/unlink/link => 404.